management/schemas/user.yaml

```diff
@@ -34,3 +36,10 @@ schemas:
         type: array
         items:
           $ref: '#/schemas/User'
+
+  AccessLevel:
+    type: string
+    enum:
+      - read
+      - edit
+      - admn
```

Comment:
Is this a typo? Not sure if it should really be `admn` or `admin` (spelled out).

Reply:
`admn` is correct. We decided on a 4-character string for the database column, and in the API we are keeping things simple and just using that value directly.